src/sass/components/_scroll.scss

```diff
@@ -1,9 +1,18 @@
-%scroll{
+%scroll-vertical{
     overflow-y: scroll;
     scrollbar-width: none;
 
     &::-webkit-scrollbar {
         display: none;
     }
 
+}
+
+%scroll-horizontal{
+    overflow-x: scroll;
+    scrollbar-width: none;
+
+    &::-webkit-scrollbar {
+        display: none;
+    }
```

Comment:
Hey, these are just placeholders they aren't components. You can place them with other placeholders

Reply:
Sure.